src/WebJobs.Script/Scale/FunctionsScaleManager.cs

```diff
@@ -16,7 +17,7 @@ namespace Microsoft.Azure.WebJobs.Script.Scale
     /// <summary>
     /// Manages scale monitoring operations.
     /// </summary>
-    public class FunctionsScaleManager
+    public class FunctionsScaleManager : IFunctionsScaleManager
```

Comment:
I'd say just remove the new interface and make GetScalersToSample virtual which will allow you to mock it in tests. Moq allows you to mock any virtual methods of a class (no interfaces required).